package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "assessment-2nd",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "jest",
+    "tdd": "jest --watch",
+    "test:coverage": "jest --collect-coverage"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/InwithSid/B5-Asessment-2nd.git"
```

Comment:
These need to be dynamic as well, based on whoever accepts the invitation of this assessment.